oclif.manifest.json.js

```
@@ -0,0 +1,11 @@
+// This file ensures the manifest is correctly loaded
+// The content is generated by oclif manifest command
+import fs from 'fs';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)